### DISCUSSION TITLE
.NET 9

### DISCUSSION THREAD

**Discussion by 0xF6** [dotnet/orleans#9226] [Q&A] [answered]
Is there any ETA for updating packages to .NET 9?

**Comment by ReubenBond** [comment, marked as answer]
I expect we will have a release today. Orleans 9.0 will not target .NET 9.0 TFM - we will keep it at .NET 8.0 to make it easier for users to upgrade. We will target 9.0 packages

**Reply by 0xF6** [reply]
Okay, thanks.
Yes, NET 8 is a LTS, we will wait for target .net9 🙃

**Reply by ReubenBond** [reply]
I don't quite understand - are you saying you will wait until Orleans targets .NET 9.0 TFM before you upgrade or are you saying you will skip .NET 9.0 & Orleans 9.0 altogether and wait for the next LTS?

You can use Orleans 9.0 with .NET Runtime/SDK 8.0 if that helps

**Reply by alrz** [reply]
> I expect we will have a release today. Orleans 9.0 will not target .NET 9.0 TFM - we will keep it at .NET 8.0 to make it easier for users to upgrade. We will target 9.0 packages

Is that a new supported setup? I don't think I remember any dotnet release doing this before.

**Reply by ReubenBond** [reply]
We are doing a similar thing with Aspire, which is a "tip" support product (the latest release is the supported release). Aspire is referencing .NET 8.0 packages, though. We may add a .NET 9 TFM (and multi-target) part way through the release if we see the need. Orleans 9.0 is up on NuGet now. Release notes: https://github.com/dotnet/orleans/releases/tag/v9.0.0
